core/src/main/scala/org/broadinstitute/dsp/workbench/welder/package.scala

```diff
@@ -75,6 +77,10 @@ package object welder {
     GsPath(basePathAndStorageLink.storageLink.cloudStorageDirectory.bucketName, fullBlobName)
   }
 
+  def hashMetadata(metadata: String): HashedMetadata = {
+    HashedMetadata(String.format("%032x", new BigInteger(1, MessageDigest.getInstance("SHA-256").digest(metadata.getBytes("UTF-8")))))
```

Comment:
I would just call this `hasString` since it has nothing to do with `metadata` in particular.
Also `String.format` may throw exception. So we should wrap it in `IO`